frontend/benefit/applicant/src/components/applications/pageContent/DecisionSummary.tsx

```diff
@@ -99,31 +99,35 @@ const DecisionSummary = ({ application }: Props): JSX.Element => {
           {t('common:applications.decision.actions.showDecision')}
         </Button>
       </$ActionContainer>
-      <$Subheading>
-        {t('common:applications.decision.headings.existingAlterations')}
-      </$Subheading>
-      <$AlterationListCount>
-        {application.alterations?.length > 0
-          ? t('common:applications.decision.alterationList.count', {
-              count: application.alterations.length,
-            })
-          : t('common:applications.decision.alterationList.empty')}
-      </$AlterationListCount>
-      <$ActionContainer>
-        {application.status === APPLICATION_STATUSES.ACCEPTED &&
-          !hasHandledTermination && (
-            <Button
-              theme="coat"
-              onClick={() =>
-                router.push(
-                  `${ROUTES.APPLICATION_ALTERATION}?id=${application.id}`
-                )
-              }
-            >
-              {t('common:applications.decision.actions.reportAlteration')}
-            </Button>
-          )}
-      </$ActionContainer>
+      {process.env.NEXT_PUBLIC_ENABLE_ALTERATION_FEATURES === 'true' && (
```

Comment:
btw, we have this handy utility function: 

https://github.com/City-of-Helsinki/yjdh/blob/ebbe9e5fd2cb21ec457845e5fe16cdfa617e655a/frontend/benefit/shared/src/utils/common.ts#L29

Reply:
I wonder why it doesn't include `(int)1`